vk/server.go

```diff
@@ -48,6 +49,12 @@ func New(opts ...OptionsModifier) *Server {
 
 // Start starts the server listening
 func (s *Server) Start() error {
+	if s.started.Load().(bool) {
```

Comment:
Were we not checking this anywhere before? 🤔

Reply:
🤷